resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,17 @@
 
   <change-notes>
     <![CDATA[
+
+0.12.0:
```

Comment:
In particular, I don't know that this assessment

> as in our first, complete and stable plugin

is really accurate.

Again, I won't fight this if you all feel strongly.